style.css

```diff
@@ -0,0 +1,7 @@
+header {
```

Comment:
- [ ] Please, try to give a class name to any HTML tag and then style it, instead of directly styling the HTML tag.